internal/cmd/tui.go

```diff
@@ -339,6 +341,41 @@ func (m tuiModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 
 			return m, tea.Quit
+
+		case "ctrl+w", "w":
```

Comment:
There is a bug where, after the second time you search, the TUI becomes totally unresponsive/ignores stdin.

Reply:
Also got this error one time, but was not able to replicate it anymore :/